src/Symfony/Component/Serializer/Tests/Normalizer/Features/CallbacksTestTrait.php

```diff
@@ -229,10 +226,7 @@ public function provideDenormalizeCallbacks(): array
             'Collect a property' => [
                 [
                     'bar' => function (array $bars) {
-                        $result = '';
-                        foreach ($bars as $bar) {
-                            $result .= $bar->bar;
-                        }
+                        $result = implode('', $bars);
 
                         return $result;
                     },
```

Comment:
Same here.

Reply:
Sorry, with "Same here." I was referring to https://github.com/symfony/symfony/pull/53119#discussion_r1429754661: The change is correct, but we can simplify the code further to a single-line arrow function.

Reply:
My bad. 🙈

Reply:
NVM, Nicolas is right. Reverting this change is correct.